### ISSUE TITLE
GPU crash with RTX A1000 on TensorFlow 2.13 — even simple Conv2D model fails

### ISSUE THREAD

**Issue by achen1959** [tensorflow/tensorflow#102065]
### Issue type

Performance

### Have you reproduced the bug with TensorFlow Nightly?

No

### Source

binary

### TensorFlow version

2.13.0

### Custom code

Yes

### OS platform and distribution

Ubuntu 24.04.3 LTS

### Python version

3.8.13

### CUDA/cuDNN version

CUDA 11.8, cuDNN 8.6

### GPU model and memory

NVIDIA RTX A1000 Laptop GPU, 6GB

### Current behavior?

Even a minimal Conv2D model causes the Python kernel to crash when run on GPU. The same model runs fine on CPU. No traceback is shown — GPU memory spikes briefly and then drops to 0 MiB. nvidia-smi shows no active processes after the crash.
Model should train normally on GPU without crashing.

### Standalone code to reproduce the issue

```shell
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Conv2D, Input
import numpy as np

model = Sequential([
    Input(shape=(32, 32, 3)),
    Conv2D(8, (3, 3), activation='relu', padding='same'),
    Conv2D(3, (3, 3), activation='sigmoid', padding='same'),
])
model.compile(optimizer='adam', loss='mse')
model.train_on_batch(np.random.rand(1, 32, 32, 3), np.random.rand(1, 32, 32, 3))
```

### Relevant log output

```shell
nvidia-smi shows ~631 MiB memory usage briefly, then drops to 0 MiB.
GPU utilization remains at 0%.
No error message in Python — kernel dies silently.
```

**Comment by CodersAcademy006** [comment]
This has the classic symptoms of a low-level driver or cuDNN incompatibility, especially given the new Ampere-based GPU and the recent Ubuntu 24.04 release. The silent crash points away from a Python-level issue and towards a problem in the underlying C++ libraries.

To help narrow this down, it would be extremely useful if the original poster could provide two key pieces of information:

1.  **The full output of the `nvidia-smi` command.** This will show the exact NVIDIA driver version, which is the most critical missing piece of information.

2.  **The result of running the same code inside the official TensorFlow Docker container.** This is the most important diagnostic step. If the code works in the container, the problem is confirmed to be in the local environment setup. If it *also* fails in the container, it points to a more serious, reproducible bug in TensorFlow itself.

The command to test this would be:
```bash
docker run --rm --gpus all -it tensorflow/tensorflow:2.13.0-gpu python -c "from tensorflow.keras.models import Sequential; from tensorflow.keras.layers import Conv2D, Input; import numpy as np; model = Sequential([Input(shape=(32, 32, 3)), Conv2D(8, (3, 3), activation='relu'), Conv2D(3, (3, 3), activation='sigmoid')]); model.compile(optimizer='adam', loss='mse'); model.train_on_batch(np.random.rand(1, 32, 32, 3), np.random.rand(1, 32, 32, 3)); print('Success!')"

**Comment by Venkat6871** [comment]
Hi @achen1959 ,
Apologies for the delay, and thanks for raising your concern here.
I tried running your code on Google Colab using TensorFlow versions 2.19.0 and 2.20.0 with GPU enabled, and the code worked fine without any crashes.

I noticed that you are using an older TensorFlow version (2.13.0). This issue might already be resolved in the newer releases. Could you please try upgrading to the latest stable TensorFlow version and check again?
Many GPU-related issues have been fixed in recent versions, so it’s always recommended to use the latest stable release for better performance and compatibility. I am attaching the Colab [gist](https://colab.sandbox.google.com/gist/Venkat6871/9b76c1ae191d8db714d9353c819cdba6/102065_tf_2-19-0-2-20-0-v-gpu.ipynb) for your reference.

Thank you!
